lbjava/src/test/java/edu/illinois/cs/cogcomp/lbjava/algo_test/main/AdaGradCLMain.java

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
Checking the folder here: https://github.com/yimingjiang/lbjava/tree/adagrad-cl/lbjava/src/test/java/edu/illinois/cs/cogcomp/lbjava/algo_test 
If the tests that you are adding are not sensitive to failures (like by assertions), we won't notice if something fails. We probably need to modify them so that they fail upon running through CI.